server/api/sample_sizes.py

```diff
@@ -99,7 +157,77 @@ def sample_sizes_for_contest(contest: Contest):
 
         else:
             assert election.audit_type == AuditType.HYBRID
-            return {}
+
+            validate_uploaded_manifests(contest)
+            validate_uploaded_cvrs(contest)
+            # TODO validate that contest choice vote counts provided by AA
+            # match with total ballots based on manifest
+
+            suite_contest = sampler_contest.from_db_contest(contest)
+
+            total_ballots = hybrid_contest_total_ballots(contest)
+            vote_counts = hybrid_contest_choice_vote_counts(contest)
+            assert vote_counts
+            non_cvr_vote_counts = {
+                choice_id: vote_count.non_cvr
+                for choice_id, vote_count in vote_counts.items()
+            }
+            cvr_vote_counts = {
+                choice_id: vote_count.cvr
+                for choice_id, vote_count in vote_counts.items()
+            }
+
+            num_previous_samples_dict = dict(
+                SampledBallotDraw.query.join(Round)
+                .filter_by(election_id=election.id)
+                .join(SampledBallot)
+                .join(Batch)
+                .group_by(Batch.has_cvrs)
+                .values(Batch.has_cvrs, func.count(SampledBallotDraw.ticket_number))
+            )
+            non_cvr_previous_samples = num_previous_samples_dict.get(False, 0)
+            cvr_previous_samples = num_previous_samples_dict.get(True, 0)
+
+            # In hybrid audits, we only store round contest results for non-CVR
+            # ballots
+            non_cvr_sample_results = (
+                {} if round_one else rounds.contest_results_by_round(contest)
+            )
+            non_cvr_stratum = suite.BallotPollingStratum(
+                total_ballots.non_cvr,
+                non_cvr_vote_counts,
+                non_cvr_sample_results,
+                non_cvr_previous_samples,
+            )
+
+            cvr_reported_results = rounds.cvrs_for_contest(contest)
```

Comment:
it occurs to me that this isn't actually used by the stratum object now that we have misstatements, so I guess I should take it out.